mysql/tests/test_query_activity.py

```diff
@@ -477,6 +478,76 @@ def _expected_dbm_job_err_tags(dbm_instance):
         'dd.internal.resource:database_instance:stubbed.hostname',
     ]
 
+@pytest.mark.integration
+@pytest.mark.usefixtures('dd_environment')
+def test_deadlocks(aggregator, dd_run_check, dbm_instance):
+    check = MySql(CHECK_NAME, {}, [dbm_instance])
+    dd_run_check(check)
+    deadlocks_start = 0
+    deadlock_metric_start =  aggregator.metrics("mysql.innodb.deadlocks")
+
+    assert len(deadlock_metric_start) == 1, "there should be one deadlock metric"
+
+    if environ.get('MYSQL_FLAVOR') == 'mariadb':
```

Comment:
we can use decorator `pytest.mark.skipif` to skip `mariadb` flavor.  For example https://github.com/DataDog/integrations-core/blob/3804613ea6173a24b00aaab585ff8328df11334d/sqlserver/tests/test_activity.py#L207

Reply:
understood, but the first part of the test is relevant for maria db. Make 2 tests ?

Reply:
yes lets make 2 tests